ChurchCRM/CRM
# Bugfix: Display custom drop-down option name in Family metadata view

#### What's this PR do?
This pull request fixes an issue in which family custom fields of type 12 (drop-down list) only render their ID value on the Family view instead of their proper name.

#### Screenshots (if appropriate)
Family Custom Drop-Down Setup:
![image](https://user-images.githubusercontent.com/23409144/115088458-325fce80-9ede-11eb-96fc-538a3e8524fc.png)

UX Before Implementation:
![image](https://user-images.githubusercontent.com/23409144/115088502-486d8f00-9ede-11eb-8e42-ee9cb70f49fa.png)

UX After Implementation:
![image](https://user-images.githubusercontent.com/23409144/115088518-515e6080-9ede-11eb-9f9d-1a4a99b5df40.png)

#### What Issues does it Close?
Closes #5242 

#### What are the relevant tickets?

#### Any background context you want to provide?
I first encountered this bug in my own implementation of ChurchCRM, finding that the v2 family view did not treat custom drop-down lists similarly to the way they are rendered in PeopleView.php.  Looking up existing bugs, I encountered #5242 which directly addressed my own experience.  

#### Where should the reviewer start?

#### How should this be manually tested?
1. Log into your instance of ChurchCRM.
2. Choose _People ->Admin->Family Custom Fields_.
3. Create a new custom field using the following criteria and choose _Add New Field_:
- _Type:_ Custom Drop-Down List
- _Name:_ Test
- _Security Option:_ bAll

4. Next to your newly-created drop-down list, choose _Edit List Options_.
5. In the _Name for New Option_ field, enter "Sample Option 1" and choose _Add New Family Custom List Option_.
6. Choose _Save Changes_.
7. Choose _People->View Active Families_.
8. Select the first family and enter _Edit_ mode.
9. In the **Custom Fields** section, select the _Test_ drop-down list and choose _Sample Option 1_.
10. Choose _Save_.
11. On the **Metadata** section of the Family view, the following text should appear:

> Test: Sample Option 1

In a failed (or original) implementation, the following text should appear:

> Test: 1

#### How should the automated tests treat this?

#### Questions:
- Is there a related website / article to substantiate / explain this change?

- Does the development wiki need an update?

- Does the user documentation wiki need an update?

- Does this add new dependencies?
  -  [X] Yes: This code change introduces the **ListOptionQuery** base class into PeopleCustomField.php.

- Does this need to add new data to the demo database
